lib/breadcrumbs_on_rails/railtie.rb

```diff
@@ -10,7 +10,10 @@ module BreadcrumbsOnRails
 
   class Railtie < Rails::Railtie
     ActiveSupport.on_load(:action_controller) do
-      include BreadcrumbsOnRails::ActionController
+      # There is no need to include breadcrumbs on an API controller
+      unless self.name == 'ActionController::API'
```

Comment:
I frankly prefer to use the code form #77, as it is much more readable and feel less hacky